source/_docs/energy/gas.markdown

```diff
@@ -29,3 +29,8 @@ We have worked with creator [Marcel Zuidwijk](https://www.zuidwijk.com) to devel
 
 ![Photo of the AI-on-the-edge-device Workflow](/images/docs/energy/ai-on-the-edge-device.jpg)
 
+#### Read the Gas Meter using a magnetometer
+
+[Diaphragm gas meters](https://en.wikipedia.org/wiki/Gas_meter#Diaphragm/bellows_meters) are the most common type of gas meter and their movement can frequently be observed with a magnetometer. The [QMC5883L](https://esphome.io/components/sensor/qmc5883l.html) is a common and inexpensive option that is supported by ESPHome. There are many posts on the forums of users having luck with this method, such as [this one](https://community.home-assistant.io/t/water-gas-meter-monitoring-via-magnetometer-sine-wave-to-pulse-issue/245904).
+
+![Photo of a typical diaphragm gas meter](https://en.wikipedia.org/wiki/File:Gas_meter.JPG)
```

Comment:
Images must be hosted from our own website.

Reply:
Just spent 45 minutes trying to figure out how to add a local picture to this PR with no luck; just went ahead and deleted it altogether.